support.md

```diff
@@ -24,9 +24,9 @@ There are also a few extra medium-specific guidelines, listed with each type of
 ### Reporting Violations
 
 Should you need to speak with someone about another Foreman member's behaviour,
-email the Community Lead, *Greg Sutcliffe* (greg.sutcliffe at gmail.com). You
-may also CC or email *Ohad Levy* (ohad.levy at gmail.com).  We'll do our best
-to listen to your issue fairly and take appropriate action.
+email the Community Lead, *Greg Sutcliffe* (greg at emeraldreverie.org, male)
+and/or *Ori Rabin* (orabin at redhat.com, female).  We'll do our best to listen
```

Comment:
@orrabin are you OK with gender in this? From a diversity point of view I think it's important that people reporting violations are made comfortable in doing so, and I've heard this information can help.